aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -376,6 +376,42 @@ impl<K: Eq + Hash + Clone + Debug + Copy> VersionedAggregators<K> {
         );
     }
 
+    /// Must be called when an snapshot (delta or derived) creation with a given ID and initial apply is observed
+    /// This should be only called when apply applies on top of different ID.
+    pub fn create_dependent_aggregator(
```

Comment:
nit: snapshot, not aggregator?

Reply:
I am using here to mean a "delayed" object, to correspond to create_aggregator which also refers to "delayed" object - as that one gets called with both aggregators and snapshots. (i.e. snapshots in Value state)

After refactoring to delayed, these two functions might be called create_delayed_slot / create_dependent_delayed_slot

(whether slot or another word for the tracking across versions, as we use entry for tracking individual version. maybe we should swap that, and call entry the whole thing attached to a key, and instance/version for a single version , but I digress)

Reply:
Yes, that would be much clearer